software/turtlebot4_desktop.md

```diff
@@ -14,37 +14,18 @@ This package is installed when following the [user computer setup instructions](
 
 ### Debian package
 
-{% tabs debian %}
-{% tab debian galactic %}
-
-```note
-The `turtlebot4_desktop` metapackage can be installed on a PC running Ubuntu Desktop 20.04 with ROS 2 Galactic.
-```
-
-To install the metapackage through apt:
-
-```bash
-sudo apt update
-sudo apt install ros-galactic-turtlebot4-desktop
-```
-
-{% endtab %}
-{% tab debian humble %}
-
-```note
-The `turtlebot4_desktop` metapackage can be installed on a PC running Ubuntu Desktop 22.04 with ROS 2 Humble.
-```
+The Turtlebot4 desktop metapackage can be installed as a debian package:
+* Ubuntu 24.04 with ROS 2 Jazzy
+* Ubuntu 22.04 with ROS 2 Humble
+* Ubuntu 20.04 with ROS 2 Galactic _end-of-life, not recommended_
 
 To install the metapackage through apt:
 
 ```bash
 sudo apt update
-sudo apt install ros-humble-turtlebot4-desktop
+sudo apt install ros-${ROS_DISTRO}-turtlebot4-desktop
```

Comment:
Can we add a note above this to ensure that they source their ros environment.

Reply:
Same as above; reverted to use tabs since we don't know for sure they even have any other ROS packages installed yet.